test/org/javarosa/core/benchmark/PopulateTreeNodeBenchmark.java

```diff
@@ -0,0 +1,75 @@
+package org.javarosa.core.benchmark;
+
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.model.instance.TreeElement;
+import org.javarosa.core.test.FormParseInit;
+import org.javarosa.core.util.PathConst;
+import org.javarosa.form.api.FormEntryController;
+import org.javarosa.xform.parse.XFormParser;
+import org.junit.Test;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.infra.Blackhole;
+import org.openjdk.jmh.results.RunResult;
+import org.openjdk.jmh.runner.NoBenchmarksException;
+import org.openjdk.jmh.runner.Runner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.javarosa.test.utils.ResourcePathHelper.r;
+import static org.junit.Assert.fail;
+
+public class PopulateTreeNodeBenchmark {
+    private static final Logger logger = LoggerFactory.getLogger(FormParserHelperParseESIBenchmark.class);
+
+    @Test
+    public void
+    launchBenchmark() throws Exception {
+        try{
+            RunResult run = new Runner(BenchmarkUtils.getJVMOptions(this.getClass().getName())).run().iterator().next();
+
+        }catch (NoBenchmarksException nbe){
+
+        }
+
+    }
+
+    @Benchmark
+    public void
+    benchmark_TreeElement_populate(TreeElementPopulateState state, Blackhole bh) throws IOException {
```

Comment:
Remove `IOException` from the signature. This method doesn't throw that exception.